__about__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = "v1.4.3"
+__version__ = "v1.5"
```

Comment:
I am following this convention for numbering (see below), and since I just added a new feature (the payload_param decorator) I bumped the version up to 1.5. Is that ok or do you want me to version down to 1.4.4?  
![image](https://cloud.githubusercontent.com/assets/17498257/19562172/a62d3236-96db-11e6-9c96-300bb4938835.png)